.github/workflows/run-integ-tests.yaml

```diff
@@ -69,7 +69,13 @@ jobs:
           set -x
           dirs_with_tests="$(
             for d in $(find * -type d -maxdepth 0 || printf ''); do
-              jq &>/dev/null -e '.scripts."integ-test"' $d/package.json && echo "$d"
+              TESTABLE_APP=$(jq &>/dev/null -e '.scripts."integ-test"' $d/package.json && echo "$d")
+              if [[ ! -z $(grep "^integ-test:" "${d}/Makefile" 2> /dev/null) ]]; then
```

Comment:
Super minor, but in case you're interested:

1. `! -z` is aka `-n`
2. you can just do `if grep -q ... ; then` (with no `[[ ... ]]]`). In that mode, `grep` doesn't actually output anything (to either stdout or stderr), but instead just exits with 0 iff any lines matched

Reply:
thanks. I used `-s` instead of `-q` to avoid printing `File not found` for `Makefile` in typescript projects.

Reply:
I didn't know about that one! nice.